doc/tutorials/setup_logical_partition_scenario_on_nuc7i7dnb.rst

```diff
@@ -0,0 +1,267 @@
+
+#. Check GRUB boot loader name. 
+
+   .. code-block:: none
+   
+      update-grub -V
+      grub-mkconfig (GRUB) 2.02~beta2-36ubuntu3.18
+
+   In this case, the GRUB loader name shall have ``GRUB`` prefix;
+#. Check or update BDF information of PCI devices of each pre-launched VM;
+   Check it in the ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``.
+
+#. Build the ACRN hypervisor and copy the artifact ``acrn.32.out`` to the
+   ``/boot`` directory:
+
+   .. code-block:: none
+   
+      $ make
+
+   .. note:: 
+
+      If image build machine is not target board (NUC7i7DNB) which boots
+      hypervisor image (acrn.32.out), need to use ``scp`` copy command to update
+      hypervisor image (acrn.32.out) on ubuntu of NUC7i7DNB.
+
+   .. note::
+
+      If image build machine is the target board (NUC7i7DNB) which boots
+      hypervisor image (acrn.32.out), need to use ``cp`` copy command to update
+      hypervisor image (acrn.32.out) on ubuntu of NUC7i7DNB.
+
+Update Ubuntu GRUB to Boot hypervisor and Load Kernel Image
+***********************************************************
+
+#. Append the following configuration to the ``/etc/grub.d/40_custom`` file:
+
+   .. code-block:: none
+
+      menuentry 'ACRN hypervisor Logical Partition Scenario' --class ubuntu --class gnu-linux --class gnu --class os $menuentry_id_option 'gnulinux-simple-e23c76ae-b06d-4a6e-ad42-46b8eedfd7d3' {
+              recordfail
+              load_video
+              gfxmode $linux_gfx_mode
+              insmod gzio
+              insmod part_gpt
+              insmod ext2
+
+              echo 'Loading hypervisor logical partition scenario ...'
+              multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+              module /boot/bzImage
+      }
+
+#. Modify the ``/etc/default/grub`` file as follows to make the GRUB menu
+   visible when booting:
+
+   .. code-block:: none
+
+      #GRUB_HIDDEN_TIMEOUT=0
+      GRUB_HIDDEN_TIMEOUT_QUIET=false
+
+#. Update grub:
+
+   .. code-block:: none
+
+      $ sudo update-grub
+
+#. Reboot the NUC7i7DNB board. Select the "ACRN hypervisor Logical Partition
+   Scenario" entry to boot the logical partition of the ACRN hypervisor on the
+   NUC7i7DNB displayer, GRUB loader will boot the hypervisor, and the hypervisor
+   will start two pre-launched VMs automatically.
+
+Logical Partition Scenario Startup Checking
+*******************************************
+
+#. Check whether ACRN hypervisor startup fails or not:
+
+   #. Login ACRN hypervisor shell from serial console.
+   #. Use the ``vm_list`` check pre-launched VMs.
+#. Check whether two pre-launched VMs startup fails or not:
+
+   #. Use the ``vm_console 0`` to switch to VM0's console.
+   #. The VM0's clearlinux OS could boot up and login in.
+   #. Use a ``Ctrl-Spacebar`` to return to the Acrn hypervisor shell.
+   #. Use the ``vm_console 1`` to switch to VM1's console.
+   #. The VM1's clearlinux OS could boot up and login in.
+
+.. note:: 
```

Comment:
I'd delete this note and replace it with a sentence:
```
Refer to the :ref:`ACRN hypervisor shell user guide <acrnshell>`
for more information about available commands.
```

Reply:
will fix at next version.